### PR TITLE
[SYCLomatic] Add comment for function calculate_max_active_wg_per_xecore()

### PR DESCRIPTION
**Pull request by zhiweij1**
Signed-off-by: Jiang, Zhiwei <zhiwei.jiang@intel.com>

### CODE REVIEW

**Review by tomflinda — APPROVED** [review]
LGTM

File: clang/test/dpct/helper_files_ref/include/util.hpp [hunk 1]
```diff
@@ -627,6 +627,8 @@ class logical_group {
 
 /// Computes the max active work-group number per Xe-Core. Ref to
```
Comment (zhimingwang36) [inline clang/test/dpct/helper_files_ref/include/util.hpp]:
```suggestion
/// This function is used for occupancy calculation,  it computes the max active work-group number per Xe-Core. Ref to
```

Comment (zhiweij1) [reply clang/test/dpct/helper_files_ref/include/util.hpp]: done

**Review by zhimingwang36 — APPROVED** [review]
LGTM

**Pull request merged**